benchmarking/nursery/transformer_wikitext/training_script.py

```diff
@@ -0,0 +1,399 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+# coding: utf-8
+import argparse
+import os
+import time
+
+# import yaml
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+try:
+    from apex import amp
+except:
+    print("Failed to import apex. You can still train with --precision {float|double}.")
+
+from pathlib import Path
+
+import data
+import model
+
+from syne_tune.report import Reporter
+from syne_tune.config_space import loguniform, uniform
+from syne_tune.constants import ST_CHECKPOINT_DIR
+
+report = Reporter()
+
+
+_config_space = {
```

Comment:
This is a pretty simple config space. Any chance this could be a bit more difficult, so we can hope to see some differences between the methods?